internal/uvm/scsi.go

```diff
@@ -395,15 +402,26 @@ func (uvm *UtilityVM) addSCSIActual(ctx context.Context, addReq *addSCSIRequest)
 		return nil, err
 	}
 
+	if existed {
+		// another mount request might be in progress, wait for it to finish and if that operation
+		// fails return that error.
+		<-sm.waitCh
```

Comment:
Should we have a select on a timeout in case we ever end up in a spot (either through an HCS change or in the guest) where we hang on attaching the device?

Reply:
Oh never mind, we have timeouts on the gcs end for rpcs and on any vmcompute calls already perfect